pkg/controller/container-runtime-config/helpers.go

```diff
@@ -253,6 +303,15 @@ func getManagedKeyCtrCfg(pool *mcfgv1.MachineConfigPool, client mcfgclientset.In
 	return fmt.Sprintf("99-%s-generated-containerruntime-%s", pool.Name, strconv.Itoa(suffixNum+1)), nil
 }
 
+func notLatestContainerRuntimeConfigInPool(ctrcfgList []mcfgv1.ContainerRuntimeConfig, cfg *mcfgv1.ContainerRuntimeConfig) bool {
+	for _, crc := range ctrcfgList {
+		if cfg.CreationTimestamp.Time.Before(crc.CreationTimestamp.Time) {
```

Comment:
> What was the issue with the automatic backport?

@yuqi-zhang  The `Compare` not defined, I changed to use the `Before()`
```
 pkg/controller/kubelet-config/helpers.go:347:27: kc.CreationTimestamp.Compare undefined (type "k8s.io/apimachinery/pkg/apis/meta/v1".Time has no field or method Compare) (typecheck)
package kubeletconfig 
```

I think this is ok, covered by thectrcfg/kcfg config_controller_test.go